### PR TITLE
older commons IO for dependency convergence

### PR DESCRIPTION
**Pull request by elharo**

### CODE REVIEW

**Review by slachiewicz — COMMENTED** [review]
Any reason to downgrade? We already have Java8 prerequisite for this plugin to run.

**Comment by sparsick** [comment]
Maybe we should upgrade the other dependencies to require Java 8

File: pom.xml [hunk 1]
```diff
@@ -112,7 +112,7 @@
     <dependency>
       <groupId>commons-io</groupId>
       <artifactId>commons-io</artifactId>
-      <version>2.8.0</version>
```
Comment (bmarwell) [inline pom.xml:115]: Please give a short reason. Either inherit it from somewhere or add a comment.
Plugins are seperated by classloaders as far as I remember, so a dependency convergence should not hurt (aside from a few KiB of jars).

**Review by bmarwell — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (elharo) [reply pom.xml:115]: check the dependency convergence report. It's not just the plugins but shared libraries such as maven-archiver.

**Comment by elharo** [comment]
Upgrading other dependencies would be a major breaking change that shoould be discussed on the mailing list and follow some analysis of how many projects and people we'd be likely to break. Most folks are on Java 8 now, but not all, and Maven is very widely used.

**Comment by bmarwell** [comment]
Assuming this plugin needs at least java 9 to work (either by toolchain or by the executing Java), I do not see the purpose of this change. No one would be affected.

**Comment by elharo** [comment]
It's not just about this plugin. It's about its own dependencies like maven-archiver that are expecting an older version of commons-io. They'll probably work with the new version, but in the rare cases when they don't it's a really nasty problem to locate.

**Review by eolivelli — CHANGES_REQUESTED** [review]
I am not sure that downgrading is good.
Which problem are you solving ?
are there test cases failing ?
is there something broken with 2.8.0 ?

do you have any error/regression ?

Plugins are running in their isolated classloader, so the dependencies of this plugin won't affect the other plugin.

Is there a JIRA case opened ?

**Review by rmannibucau — APPROVED** [review]
Better to either drop this dependency which is not that justified on j8 or keep it up to date so +1. Downgrading will just explode later, we should never do it.

**Pull request closed without merging**